src/main/java/seedu/quotesify/Quotesify.java

```diff
@@ -62,16 +74,27 @@ public void runLoopUntilExitCommand() {
         }
     }
 
+    /**
+     * Runs the program until termination.
+     */
     public void run() {
         start();
         runLoopUntilExitCommand();
         exit();
     }
 
+    /**
+     * Program main method.
+     *
+     * @param args user specified arguments
+     */
```

Comment:
Based on what I see from the coding standard, the first word after the variable name should be capitalized. For example in this case the "user" word. This applies for param, return, throws, etc.

Also, every line of the JavaDocs should end with a period. Think you missed out some, might want to double check.

Reply:
Alright noted!